pkg/actions/service_test.go

```diff
@@ -290,12 +290,6 @@ func TestDisableHooksRun(t *testing.T) {
 				t.Fatalf("Run() failed with err=%s", err)
 			}
 
-			// update commit using post event record
-			err = actionsService.UpdateCommitID(ctx, record.RepositoryID.String(), record.StorageNamespace.String(), record.RunID, "commit1")
```

Comment:
Why did you decide to remove it?

Reply:
Removed due to previous changes in behavior which were decided. Added it back